### PR TITLE
[jellyfin] Update to 10.8.1 and fix typo in Chart.yaml homepage URL

### PR DESCRIPTION
**Pull request by mcronce**
**Description of the change**
* Updates Jellyfin to 10.8.1
* Fixes typo in Chart.yaml homepage url (capital J instead of lower-case)

**Benefits**
* Updated version
* artifacthub.io homepage link will go to the right place now :)

**Possible drawbacks**
N/A

**Applicable issues**
N/A

**Additional information**
Not sure if 10.7.7 to 10.8.1 would warrant bumping the chart minor version instead of patch, can change it if so

**Checklist** 
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Chart `artifacthub.io/changes` changelog annotation has been updated in `Chart.yaml`. See [Artifact Hub documentation](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations) for more info.
- [x] Variables have been documented in the `values.yaml` file.

### CODE REVIEW

**Comment by mcronce** [comment]
@onedr0p @bjw-s @billimek @carpenike @Truxnell sorry for the @ spam.  It _looks like_ you're all intended to be code owners, but `.github/CODEOWNERS` was renamed to `.github/values.yaml` in 2a111668 - which I'm guessing was not intentional?

**Comment by bjw-s** [comment]
> @onedr0p @bjw-s @billimek @carpenike @Truxnell sorry for the @ spam. It _looks like_ you're all intended to be code owners, but `.github/CODEOWNERS` was renamed to `.github/values.yaml` in [2a11166](https://github.com/k8s-at-home/charts/commit/2a11166887b60747dd8d1eaa700e80a8e23663f1) - which I'm guessing was not intentional?

Thanks for pointing this out! I've fixed it in the `master` branch.
I'll try to take a look at the PR later this evening / tomorrow

File: charts/stable/jellyfin/values.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ image:
   # -- image repository
   repository: jellyfin/jellyfin
   # -- image tag
-  tag: 10.7.7
+  tag: 10.8.1
```
Comment (bjw-s) [inline charts/stable/jellyfin/values.yaml:12]:
```suggestion
  # @default -- chart.appVersion
  tag:
```
By doing this we will only have to change the `appVersion` field in the Chart.yaml file in the future.

**Review by bjw-s — CHANGES_REQUESTED** [review]
Thanks a lot for the PR! Just a small change to the values.yaml requested and then it's good to go 🙂

File: charts/stable/jellyfin/values.yaml (hunk 1 above)
Comment (mcronce) [reply charts/stable/jellyfin/values.yaml]: Done - TIL that's a thing, cool feature

Comment (mcronce) [reply charts/stable/jellyfin/values.yaml]: Should the values table in `README.md` get an update as well?

Comment (bjw-s) [reply charts/stable/jellyfin/values.yaml]: Normally that will be taken care of automatically when the chart gets merged :)

Comment (mcronce) [reply charts/stable/jellyfin/values.yaml]: Cool - in that case, this should be good to go

**Review by bjw-s — DISMISSED** [review]

**Comment by mcronce** [comment]
@bjw-s sorry, bad timing ;) I just pushed a fix for the merge conflicts

**Comment by bjw-s** [comment]
> @bjw-s sorry, bad timing ;) I just pushed a fix for the merge conflicts

haha np :)

**Review by bjw-s — APPROVED** [review]

**Pull request merged**